frontend/app/page.tsx

```diff
@@ -6,10 +6,12 @@ import { logo } from './content/birthdayAppLogo';
 import { ethers } from 'ethers';
 
 function Birthday() {
-    const [input, setInput] = useState('0x3a0F761126B034e3031d3C934eDA62251A07D7f1');
+    const [input, setInput] = useState('0x289B7B7A15f0B696f1196017920eA895762f011c');
```

Comment:
What is this address? should not be hardcoded. Create a variable, with clear naming